### DISCUSSION TITLE
Shortcut for comment/uncomment in Yaml Editor

### DISCUSSION THREAD

**Discussion by xavier-GitHub76** [louislam/dockge#259] [Feature Request]
### ⚠️ Please verify that this feature request has NOT been suggested before.

- [X] I checked and didn't find similar feature request

### 🏷️ Feature Request Type

UI Feature

### 🔖 Feature description

Hello,
it could be pratical to can use a shortcut for comment/uncomment lines in Yaml Editor
Best regards

### ✔️ Solution

CTRL+Shift+: could be used for the shortcut (like Visual studio or Portainer)

**Comment by andersmmg** [comment]
This would be really nice for me as well, #478 would need to be fixed as well since it would affect this. Also, CTRL+/ would be a useful shortcut for this as many editors use this

**Comment by hockeygoalie35** [comment]
+1, missing this as I migrated off of portainer.
